
   _____ ____  _    _  ____  _____ _______ ______
  / ____/ __ \| |  | |/ __ \|  __ \__   __|  ____|
 | |   | |  | | |__| | |  | | |__) | | |  | |__
 | |   | |  | |  __  | |  | |  _  /  | |  |  __|
 | |___| |__| | |  | | |__| | | \ \  | |  | |____
  \_____\____/|_|  |_|\____/|_|  \_\ |_|  |______|
    
     APPLICATION ID : demo
          NODE NAME : node
         TRANSPORTS : http

       TOP COMPOSER : True

          HTTP PORT : 0
         SHELL PORT : 0

   COMPOSITION FILE : composition.js
         AUTO START : True


    COHORTE VERSION : 1.2.1_20170707-092230_release
       COHORTE HOME : /Users/apisu/Documents/tests/cohorte/cohorte-test
       COHORTE BASE : /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node
       COHORTE DATA : /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/data
 PYTHON INTERPRETER : python (3.6.2)
        PYTHON PATH : /Applications/eclipse-jee-neon-3-Python/eclipse-jee-neon-3-Python/eclipse-jee-neon-3-Python.app/Contents/Eclipse/plugins/org.python.pydev_5.7.0.201704111357/pysrc/pydev_sitecustomize
			/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6
			/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/lib-dynload
			/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages
			/Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/repo
			/Users/apisu/Documents/tests/cohorte/cohorte-test/repo
  OS PATH SEPARATOR : :
           LOG FILE : /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/var/forker.log
           

            You should have Python 3.4 to launch Java isolates!
            Your Python version is not yet supported!
  - call boot.py with args: [['-t', '--console']]2017-10-05 12:12:13,424:INFO    :root                :MainThread     : Root log configured. (boot.configure_logger() @477)
2017-10-05 12:12:13,505:INFO    :cohorte.config.includer:MainThread     :  validating (includer.validate() @579)
2017-10-05 12:12:13,505:INFO    :cohorte.config.includer:MainThread     : validated (includer.validate() @581)
2017-10-05 12:12:13,525:INFO    :cohorte.config.includer:MainThread     : _getContent conf/boot-forker.js (includer._get_content() @375)
2017-10-05 12:12:13,525:INFO    :cohorte.config.includer:MainThread     : filename conf/boot-forker.js (includer._init_filename() @341)
2017-10-05 12:12:13,527:INFO    :cohorte.config.includer:MainThread     : read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/boot-forker.js (includer._read_contents_file() @263)
2017-10-05 12:12:13,528:INFO    :cohorte.config.includer:MainThread     : read file /Users/apisu/Documents/tests/cohorte/cohorte-test/conf/boot-forker.js (includer._read_contents_file() @263)
2017-10-05 12:12:13,530:INFO    :cohorte.config.includer:MainThread     : read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/boot-forker.js (includer._read_contents_file() @263)
2017-10-05 12:12:13,538:INFO    :cohorte.config.includer:MainThread     : _getContent conf/boot-common-py.js (includer._get_content() @375)
2017-10-05 12:12:13,539:INFO    :cohorte.config.includer:MainThread     : filename conf/boot-common-py.js (includer._init_filename() @341)
2017-10-05 12:12:13,540:INFO    :cohorte.config.includer:MainThread     : read file /Users/apisu/Documents/tests/cohorte/cohorte-test/conf/boot-common-py.js (includer._read_contents_file() @263)
2017-10-05 12:12:13,545:INFO    :cohorte.config.includer:MainThread     : _getContent conf/boot-common.js (includer._get_content() @375)
2017-10-05 12:12:13,546:INFO    :cohorte.config.includer:MainThread     : filename conf/boot-common.js (includer._init_filename() @341)
2017-10-05 12:12:13,547:INFO    :cohorte.config.includer:MainThread     : read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/boot-common.js (includer._read_contents_file() @263)
2017-10-05 12:12:13,548:INFO    :cohorte.config.includer:MainThread     : read file /Users/apisu/Documents/tests/cohorte/cohorte-test/conf/boot-common.js (includer._read_contents_file() @263)
2017-10-05 12:12:13,549:INFO    :cohorte.config.includer:MainThread     : read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/boot-common.js (includer._read_contents_file() @263)
2017-10-05 12:12:13,553:INFO    :cohorte.config.includer:MainThread     : _getContent conf/herald/python-core.js (includer._get_content() @375)
2017-10-05 12:12:13,554:INFO    :cohorte.config.includer:MainThread     : filename conf/herald/python-core.js (includer._init_filename() @341)
2017-10-05 12:12:13,557:INFO    :cohorte.config.includer:MainThread     : read file /Users/apisu/Documents/tests/cohorte/cohorte-test/conf/herald/python-core.js (includer._read_contents_file() @263)
2017-10-05 12:12:13,564:INFO    :cohorte.config.includer:MainThread     : _getContent conf/herald/python-transport.js (includer._get_content() @375)
2017-10-05 12:12:13,564:INFO    :cohorte.config.includer:MainThread     : filename conf/herald/python-transport.js (includer._init_filename() @341)
2017-10-05 12:12:13,566:INFO    :cohorte.config.includer:MainThread     : read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/herald/python-transport.js (includer._read_contents_file() @263)
2017-10-05 12:12:13,567:INFO    :cohorte.config.includer:MainThread     : read file /Users/apisu/Documents/tests/cohorte/cohorte-test/conf/herald/python-transport.js (includer._read_contents_file() @263)
2017-10-05 12:12:13,569:INFO    :cohorte.config.includer:MainThread     : read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/herald/python-transport.js (includer._read_contents_file() @263)
2017-10-05 12:12:13,570:INFO    :cohorte.config.includer:MainThread     : _getContent conf/herald/python-http.js (includer._get_content() @375)
2017-10-05 12:12:13,571:INFO    :cohorte.config.includer:MainThread     : filename conf/herald/python-http.js (includer._init_filename() @341)
2017-10-05 12:12:13,572:INFO    :cohorte.config.includer:MainThread     : read file /Users/apisu/Documents/tests/cohorte/cohorte-test/conf/herald/python-http.js (includer._read_contents_file() @263)
2017-10-05 12:12:13,577:INFO    :cohorte.config.includer:MainThread     : _getContent conf/python-common-http.js (includer._get_content() @375)
2017-10-05 12:12:13,577:INFO    :cohorte.config.includer:MainThread     : filename conf/python-common-http.js (includer._init_filename() @341)
2017-10-05 12:12:13,579:INFO    :cohorte.config.includer:MainThread     : read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/python-common-http.js (includer._read_contents_file() @263)
2017-10-05 12:12:13,580:INFO    :cohorte.config.includer:MainThread     : read file /Users/apisu/Documents/tests/cohorte/cohorte-test/conf/python-common-http.js (includer._read_contents_file() @263)
2017-10-05 12:12:13,583:INFO    :cohorte.config.includer:MainThread     : read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/python-common-http.js (includer._read_contents_file() @263)
2017-10-05 12:12:13,589:INFO    :cohorte.config.includer:MainThread     : _getContent conf/composer/python-node.js (includer._get_content() @375)
2017-10-05 12:12:13,589:INFO    :cohorte.config.includer:MainThread     : filename conf/composer/python-node.js (includer._init_filename() @341)
2017-10-05 12:12:13,591:INFO    :cohorte.config.includer:MainThread     : read file /Users/apisu/Documents/tests/cohorte/cohorte-test/conf/composer/python-node.js (includer._read_contents_file() @263)
2017-10-05 12:12:13,598:INFO    :cohorte.boot.loaders.forker:MainThread     : Loading a forker with the following properties:
* Home.....: /Users/apisu/Documents/tests/cohorte/cohorte-test
* Base.....: /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node
* Node UID.: a43735fc-1edf-46ce-836b-e302f3af0ac3
* Node Name: node
* Node Data: /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/data
* UID......: f0303e67-3494-485f-962a-8597cce4a239 (forker.load() @249)
2017-10-05 12:12:13,907:INFO    :pelix.shell.core    :MainThread     : Shell services registered (core.start() @813)
2017-10-05 12:12:14,046:INFO    :pelix-http-service  :MainThread     : Starting HTTP server: [0.0.0.0]:0 ... (basic.log() @802)
2017-10-05 12:12:14,058:INFO    :pelix-http-service  :MainThread     : HTTP server started: [0.0.0.0]:63008 (basic.log() @802)
2017-10-05 12:12:14,070:INFO    :herald.core         :Herald-Bind    : No peer in group all (core.fire_group() @711)
2017-10-05 12:12:14,074:INFO    :proxy.proxy         :Herald-Bind    : HTTP Service Proxy validated (proxy.validate() @245)
2017-10-05 12:12:14,075:INFO    :proxy.proxy         :Herald-Bind    : Bound to / (proxy.bound_to() @260)
2017-10-05 12:12:14,076:INFO    :pelix.shell.remote  :MainThread     : RemoteShell validated on port: 63009 (remote.validate() @373)
2017-10-05 12:12:14,080:INFO    :cohorte.debug.servlet:MainThread     : Debug servlet Ready (servlet.validate() @362)
2017-10-05 12:12:14,082:INFO    :debug.agent         :MainThread     : Debug agent Ready (agent.validate() @527)
2017-10-05 12:12:14,090:INFO    :cohorte.monitor.status:MainThread     : Monitor status validated (status.validate() @184)
2017-10-05 12:12:14,535:INFO    :cohorte.repositories.python.modules:MainThread     : Loading repository from file system... (modules.validate() @618)
2017-10-05 12:12:14,596:INFO    :cohorte.monitor.basic:Thread-6       : Monitor core validated (basic.validate() @479)
2017-10-05 12:12:14,607:INFO    :cohorte.config.includer:Thread-6       : _getContent conf/autorun_isolates.js (includer._get_content() @375)
2017-10-05 12:12:14,615:INFO    :cohorte.config.includer:Thread-6       : filename conf/autorun_isolates.js (includer._init_filename() @341)
2017-10-05 12:12:14,620:INFO    :cohorte.config.includer:Top-Composer-Loader: _getContent conf/composer/python-top.js (includer._get_content() @375)
2017-10-05 12:12:14,624:INFO    :cohorte.config.includer:Top-Composer-Loader: filename conf/composer/python-top.js (includer._init_filename() @341)
2017-10-05 12:12:14,626:INFO    :cohorte.config.includer:Top-Composer-Loader: read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/composer/python-top.js (includer._read_contents_file() @263)
2017-10-05 12:12:14,628:INFO    :cohorte.config.includer:Top-Composer-Loader: read file /Users/apisu/Documents/tests/cohorte/cohorte-test/conf/composer/python-top.js (includer._read_contents_file() @263)
2017-10-05 12:12:14,630:INFO    :cohorte.config.includer:Top-Composer-Loader: read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/composer/python-top.js (includer._read_contents_file() @263)
2017-10-05 12:12:14,809:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Users/apisu/Documents/tests/cohorte/cohorte-test/repo/cohorte/config/common.py: 'ascii' codec can't decode byte 0xc3 in position 97: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:14,813:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Users/apisu/Documents/tests/cohorte/cohorte-test/repo/cohorte/config/includer.py: 'ascii' codec can't decode byte 0xc3 in position 251: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:14,929:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Users/apisu/Documents/tests/cohorte/cohorte-test/repo/cohorte/utils/vote.py: 'ascii' codec can't decode byte 0xc3 in position 1634: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:14,935:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Users/apisu/Documents/tests/cohorte/cohorte-test/repo/cohorte/vote/beans.py: 'ascii' codec can't decode byte 0xc3 in position 1927: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:15,271:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Users/apisu/Documents/tests/cohorte/cohorte-test/repo/requests/status_codes.py: 'ascii' codec can't decode byte 0xe2 in position 323: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:16,041:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Users/apisu/Documents/tests/cohorte/cohorte-test/repo/sleekxmpp/xmlstream/scheduler.py: 'ascii' codec can't decode byte 0xe2 in position 5814: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:16,410:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/functools.py: 'ascii' codec can't decode byte 0xc5 in position 300: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:16,412:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/getopt.py: 'ascii' codec can't decode byte 0xc3 in position 829: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:16,426:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/heapq.py: 'ascii' codec can't decode byte 0xc3 in position 1401: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:16,737:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/shlex.py: 'ascii' codec can't decode byte 0xc3 in position 1333: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:16,786:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/sre_compile.py: 'ascii' codec can't decode byte 0xc4 in position 697: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:17,385:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/distutils/command/bdist_msi.py: 'ascii' codec can't decode byte 0xc3 in position 39: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:17,433:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/distutils/tests/test_archive_util.py: 'ascii' codec can't decode byte 0xc3 in position 2447: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:17,488:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/distutils/tests/test_msvccompiler.py: 'ascii' codec can't decode byte 0xe1 in position 3229: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:17,489:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/distutils/tests/test_register.py: 'ascii' codec can't decode byte 0xc3 in position 6331: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:17,585:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/email/message.py: 'ascii' codec can't decode byte 0xc3 in position 20842: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:17,769:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/encodings/koi8_t.py: 'ascii' codec can't decode byte 0xd0 in position 77: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:17,795:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/encodings/punycode.py: 'ascii' codec can't decode byte 0xc3 in position 85: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:18,115:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/idlelib/idle_test/test_hyperparser.py: 'ascii' codec can't decode byte 0xc3 in position 1034: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:18,373:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/lib2to3/tests/test_main.py: 'ascii' codec can't decode byte 0xc3 in position 1183: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:18,499:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/pydoc_data/topics.py: 'ascii' codec can't decode byte 0xe2 in position 178022: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:18,558:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/asn1crypto/core.py: 'ascii' codec can't decode byte 0xe2 in position 17494: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:18,852:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/chardet/langturkishmodel.py: 'ascii' codec can't decode byte 0xc3 in position 411: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:18,877:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/commentjson/commentjson.py: Error parsing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/commentjson/commentjson.py: invalid syntax (commentjson.py, line 58) (modules.add_directory() @384)
2017-10-05 12:12:19,068:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/idna/uts46data.py: 'ascii' codec can't decode byte 0xcc in position 3234: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:19,101:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/jsoncomment/__init__.py: 'ascii' codec can't decode byte 0xef in position 0: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:19,102:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/jsoncomment/package/__init__.py: 'ascii' codec can't decode byte 0xef in position 0: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:19,103:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/jsoncomment/package/comments.py: 'ascii' codec can't decode byte 0xef in position 0: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:19,104:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/jsoncomment/package/wrapper.py: 'ascii' codec can't decode byte 0xef in position 0: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:19,168:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_vendor/appdirs.py: 'ascii' codec can't decode byte 0xc8 in position 129: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:19,531:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_vendor/lockfile/pidlockfile.py: 'ascii' codec can't decode byte 0xc2 in position 56: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:19,549:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_vendor/pkg_resources/__init__.py: 'ascii' codec can't decode byte 0xef in position 63267: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:19,572:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_vendor/progress/bar.py: 'ascii' codec can't decode byte 0xe2 in position 1610: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:19,573:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_vendor/progress/counter.py: 'ascii' codec can't decode byte 0xe2 in position 1191: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:19,574:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_vendor/progress/spinner.py: 'ascii' codec can't decode byte 0xe2 in position 1116: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:19,603:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_vendor/requests/status_codes.py: 'ascii' codec can't decode byte 0xe2 in position 323: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:19,899:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_vendor/webencodings/__init__.py: 'ascii' codec can't decode byte 0xe2 in position 1683: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:19,903:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_vendor/webencodings/tests.py: 'ascii' codec can't decode byte 0xc2 in position 1006: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:19,989:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pkg_resources/__init__.py: 'ascii' codec can't decode byte 0xef in position 63338: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:19,991:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pkg_resources/_vendor/appdirs.py: 'ascii' codec can't decode byte 0xc8 in position 129: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:20,271:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/requests/certs.py: 'ascii' codec can't decode byte 0xe2 in position 165: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:20,311:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/requests/status_codes.py: 'ascii' codec can't decode byte 0xe2 in position 323: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:20,606:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/sqlite3/__init__.py: 'ascii' codec can't decode byte 0xc3 in position 80: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:20,607:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/sqlite3/dbapi2.py: 'ascii' codec can't decode byte 0xc3 in position 85: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:20,609:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/sqlite3/test/dbapi.py: 'ascii' codec can't decode byte 0xe4 in position 120: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:20,611:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/sqlite3/test/factory.py: 'ascii' codec can't decode byte 0xe4 in position 138: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:20,612:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/sqlite3/test/hooks.py: 'ascii' codec can't decode byte 0xe4 in position 132: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:20,613:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/sqlite3/test/regression.py: 'ascii' codec can't decode byte 0xe4 in position 123: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:20,614:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/sqlite3/test/transactions.py: 'ascii' codec can't decode byte 0xe4 in position 118: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:20,615:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/sqlite3/test/types.py: 'ascii' codec can't decode byte 0xe4 in position 127: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:20,616:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/sqlite3/test/userfunctions.py: 'ascii' codec can't decode byte 0xe4 in position 184: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:20,682:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/bad_coding2.py: 'ascii' codec can't decode byte 0xef in position 0: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:20,683:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/badsyntax_3131.py: 'ascii' codec can't decode byte 0xe2 in position 24: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:20,690:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/badsyntax_pep3120.py: 'ascii' codec can't decode byte 0xf6 in position 8: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:20,853:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/pickletester.py: 'ascii' codec can't decode byte 0xcf in position 25722: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:21,191:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_builtin.py: 'ascii' codec can't decode byte 0xc3 in position 7137: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:21,301:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_codecs.py: 'ascii' codec can't decode byte 0xc4 in position 68706: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:21,362:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_configparser.py: 'ascii' codec can't decode byte 0xe7 in position 53576: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:21,416:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_csv.py: 'ascii' codec can't decode byte 0xc3 in position 43698: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:21,609:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_difflib.py: 'ascii' codec can't decode byte 0xc4 in position 4253: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:21,620:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_doctest.py: 'ascii' codec can't decode byte 0xc3 in position 57444: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:21,620:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_doctest2.py: 'ascii' codec can't decode byte 0xd0 in position 232: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:21,888:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_getpass.py: 'ascii' codec can't decode byte 0xc3 in position 2515: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:21,916:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_hash.py: 'ascii' codec can't decode byte 0xc3 in position 7077: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:21,929:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_hmac.py: 'ascii' codec can't decode byte 0xc3 in position 19518: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:21,929:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_html.py: 'ascii' codec can't decode byte 0xc2 in position 3565: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:21,991:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_httpservers.py: 'ascii' codec can't decode byte 0xc3 in position 3433: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:21,993:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_imaplib.py: 'ascii' codec can't decode byte 0xc3 in position 10998: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:22,220:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_linecache.py: 'ascii' codec can't decode byte 0xc3 in position 2143: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:22,224:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_locale.py: 'ascii' codec can't decode byte 0xc3 in position 13683: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:22,292:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_mailbox.py: 'ascii' codec can't decode byte 0xc3 in position 2975: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:22,411:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_nntplib.py: 'ascii' codec can't decode byte 0xc3 in position 22722: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:22,414:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_ntpath.py: 'ascii' codec can't decode byte 0xc4 in position 3047: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:22,515:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_pathlib.py: 'ascii' codec can't decode byte 0xc3 in position 56769: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:22,611:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_pprint.py: 'ascii' codec can't decode byte 0xc3 in position 33236: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:22,629:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_pydoc.py: 'ascii' codec can't decode byte 0xd0 in position 758: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:22,665:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_re.py: 'ascii' codec can't decode byte 0xc2 in position 11931: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:22,666:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_readline.py: 'ascii' codec can't decode byte 0xc3 in position 3577: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:22,699:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_sax.py: 'ascii' codec can't decode byte 0xc3 in position 11262: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:22,763:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_shelve.py: 'ascii' codec can't decode byte 0xc3 in position 3227: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:22,764:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_shlex.py: 'ascii' codec can't decode byte 0xc3 in position 1835: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:22,799:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_smtpd.py: 'ascii' codec can't decode byte 0xc5 in position 36644: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:22,800:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_smtplib.py: 'ascii' codec can't decode byte 0xc3 in position 40614: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:22,804:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_socket.py: 'ascii' codec can't decode byte 0xd0 in position 48524: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:22,812:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_source_encoding.py: 'ascii' codec can't decode byte 0xf0 in position 319: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:22,815:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_ssl.py: 'ascii' codec can't decode byte 0xc3 in position 19947: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:22,872:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_string_literals.py: 'ascii' codec can't decode byte 0xc3 in position 8731: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:23,068:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_tokenize.py: 'ascii' codec can't decode byte 0xc3 in position 23377: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:23,162:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_unicode.py: 'ascii' codec can't decode byte 0xc2 in position 33736: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:23,166:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_unicode_identifiers.py: 'ascii' codec can't decode byte 0xc3 in position 122: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:23,265:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_utf8source.py: 'ascii' codec can't decode byte 0xd0 in position 204: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:23,303:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_winconsoleio.py: 'ascii' codec can't decode byte 0xcf in position 4165: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:23,304:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_winreg.py: 'ascii' codec can't decode byte 0xe6 in position 2081: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:23,322:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_xml_etree.py: 'ascii' codec can't decode byte 0xc3 in position 103036: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:23,327:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_xmlrpc.py: 'ascii' codec can't decode byte 0xf0 in position 9269: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:23,423:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/encoded_modules/__init__.py: 'ascii' codec can't decode byte 0xc3 in position 994: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:23,424:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/encoded_modules/module_iso_8859_1.py: 'ascii' codec can't decode byte 0xe9 in position 87: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:23,425:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/encoded_modules/module_koi8_r.py: 'ascii' codec can't decode byte 0xf0 in position 59: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:23,659:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_email/test__encoded_words.py: 'ascii' codec can't decode byte 0xc3 in position 2387: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:23,660:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_email/test__header_value_parser.py: 'ascii' codec can't decode byte 0xc3 in position 6726: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:23,663:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_email/test_contentmanager.py: 'ascii' codec can't decode byte 0xc3 in position 6145: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:23,666:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_email/test_email.py: 'ascii' codec can't decode byte 0xc3 in position 25687: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:23,667:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_email/test_generator.py: 'ascii' codec can't decode byte 0xc3 in position 7284: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:23,668:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_email/test_headerregistry.py: 'ascii' codec can't decode byte 0xc3 in position 5594: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:23,670:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_email/test_message.py: 'ascii' codec can't decode byte 0xc3 in position 1061: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:23,692:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_importlib/test_abc.py: 'ascii' codec can't decode byte 0xc3 in position 26291: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:23,709:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_importlib/test_util.py: 'ascii' codec can't decode byte 0xc3 in position 419: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:23,828:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/test/test_warnings/__init__.py: 'ascii' codec can't decode byte 0xc3 in position 45596: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:23,996:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/tkinter/test/test_ttk/test_functions.py: 'ascii' codec can't decode byte 0xce in position 2083: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,252:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/urllib/parse.py: 'ascii' codec can't decode byte 0xe2 in position 1680: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,397:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/asn1crypto/core.py: 'ascii' codec can't decode byte 0xe2 in position 17494: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,411:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/chardet/langturkishmodel.py: 'ascii' codec can't decode byte 0xc3 in position 411: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,416:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/commentjson/commentjson.py: Error parsing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/commentjson/commentjson.py: invalid syntax (commentjson.py, line 58) (modules.add_directory() @384)
2017-10-05 12:12:24,435:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/idna/uts46data.py: 'ascii' codec can't decode byte 0xcc in position 3234: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,444:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/jsoncomment/__init__.py: 'ascii' codec can't decode byte 0xef in position 0: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,446:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/jsoncomment/package/__init__.py: 'ascii' codec can't decode byte 0xef in position 0: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,446:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/jsoncomment/package/comments.py: 'ascii' codec can't decode byte 0xef in position 0: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,447:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/jsoncomment/package/wrapper.py: 'ascii' codec can't decode byte 0xef in position 0: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,450:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_vendor/appdirs.py: 'ascii' codec can't decode byte 0xc8 in position 129: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,474:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_vendor/lockfile/pidlockfile.py: 'ascii' codec can't decode byte 0xc2 in position 56: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,478:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_vendor/pkg_resources/__init__.py: 'ascii' codec can't decode byte 0xef in position 63267: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,479:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_vendor/progress/bar.py: 'ascii' codec can't decode byte 0xe2 in position 1610: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,480:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_vendor/progress/counter.py: 'ascii' codec can't decode byte 0xe2 in position 1191: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,481:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_vendor/progress/spinner.py: 'ascii' codec can't decode byte 0xe2 in position 1116: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,485:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_vendor/requests/status_codes.py: 'ascii' codec can't decode byte 0xe2 in position 323: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,504:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_vendor/webencodings/__init__.py: 'ascii' codec can't decode byte 0xe2 in position 1683: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,505:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_vendor/webencodings/tests.py: 'ascii' codec can't decode byte 0xc2 in position 1006: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,517:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pkg_resources/__init__.py: 'ascii' codec can't decode byte 0xef in position 63338: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,518:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pkg_resources/_vendor/appdirs.py: 'ascii' codec can't decode byte 0xc8 in position 129: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,529:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/requests/certs.py: 'ascii' codec can't decode byte 0xe2 in position 165: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,531:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/requests/status_codes.py: 'ascii' codec can't decode byte 0xe2 in position 323: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,586:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Users/apisu/Documents/tests/cohorte/cohorte-test/repo/cohorte/config/common.py: 'ascii' codec can't decode byte 0xc3 in position 97: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,587:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Users/apisu/Documents/tests/cohorte/cohorte-test/repo/cohorte/config/includer.py: 'ascii' codec can't decode byte 0xc3 in position 251: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,607:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Users/apisu/Documents/tests/cohorte/cohorte-test/repo/cohorte/utils/vote.py: 'ascii' codec can't decode byte 0xc3 in position 1634: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,610:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Users/apisu/Documents/tests/cohorte/cohorte-test/repo/cohorte/vote/beans.py: 'ascii' codec can't decode byte 0xc3 in position 1927: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,640:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Users/apisu/Documents/tests/cohorte/cohorte-test/repo/requests/status_codes.py: 'ascii' codec can't decode byte 0xe2 in position 323: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,722:WARNING :cohorte.repositories.python.modules:MainThread     : Error analyzing /Users/apisu/Documents/tests/cohorte/cohorte-test/repo/sleekxmpp/xmlstream/scheduler.py: 'ascii' codec can't decode byte 0xe2 in position 5814: ordinal not in range(128) (modules.add_directory() @384)
2017-10-05 12:12:24,808:INFO    :cohorte.composer.top.composer:Top-Composer-Loader: Auto-start composition: True (composer.validate() @126)
2017-10-05 12:12:24,808:INFO    :cohorte.config.includer:Top-Composer-Loader: _getContent conf/composition.js (includer._get_content() @375)
2017-10-05 12:12:24,808:INFO    :cohorte.config.includer:Top-Composer-Loader: filename conf/composition.js (includer._init_filename() @341)
2017-10-05 12:12:24,811:INFO    :cohorte.config.includer:Top-Composer-Loader: read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/composition.js (includer._read_contents_file() @263)
2017-10-05 12:12:24,829:INFO    :cohorte.config.includer:Top-Composer-Loader: read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/composition.js (includer._read_contents_file() @263)
2017-10-05 12:12:24,837:INFO    :cohorte.composer.top.composer:Top-Composer-Loader: Loading composition... (composer.validate() @133)
2017-10-05 12:12:24,838:INFO    :cohorte.config.includer:Top-Composer-Loader: _getContent conf/web.js (includer._get_content() @375)
2017-10-05 12:12:24,839:INFO    :cohorte.config.includer:Top-Composer-Loader: filename conf/web.js (includer._init_filename() @341)
2017-10-05 12:12:24,844:INFO    :cohorte.config.includer:Top-Composer-Loader: _getContent conf/isolate_web.js (includer._get_content() @375)
2017-10-05 12:12:24,844:INFO    :cohorte.config.includer:Top-Composer-Loader: filename conf/isolate_web.js (includer._init_filename() @341)
2017-10-05 12:12:24,847:INFO    :cohorte.config.includer:Top-Composer-Loader: read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/isolate_web.js (includer._read_contents_file() @263)
2017-10-05 12:12:24,851:INFO    :cohorte.config.includer:Top-Composer-Loader: read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/isolate_web.js (includer._read_contents_file() @263)
2017-10-05 12:12:24,857:INFO    :cohorte.config.includer:Top-Composer-Loader: _getContent conf/components1.js (includer._get_content() @375)
2017-10-05 12:12:24,859:INFO    :cohorte.config.includer:Top-Composer-Loader: filename conf/components1.js (includer._init_filename() @341)
2017-10-05 12:12:24,863:INFO    :cohorte.config.includer:Top-Composer-Loader: _getContent conf/isolate_components1.js (includer._get_content() @375)
2017-10-05 12:12:24,863:INFO    :cohorte.config.includer:Top-Composer-Loader: filename conf/isolate_components1.js (includer._init_filename() @341)
2017-10-05 12:12:24,866:INFO    :cohorte.config.includer:Top-Composer-Loader: read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/isolate_components1.js (includer._read_contents_file() @263)
2017-10-05 12:12:24,869:INFO    :cohorte.config.includer:Top-Composer-Loader: read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/isolate_components1.js (includer._read_contents_file() @263)
2017-10-05 12:12:24,872:INFO    :cohorte.config.includer:Top-Composer-Loader: _getContent conf/components2.js (includer._get_content() @375)
2017-10-05 12:12:24,873:INFO    :cohorte.config.includer:Top-Composer-Loader: filename conf/components2.js (includer._init_filename() @341)
2017-10-05 12:12:24,879:INFO    :cohorte.config.includer:Top-Composer-Loader: _getContent conf/isolate_components2.js (includer._get_content() @375)
2017-10-05 12:12:24,880:INFO    :cohorte.config.includer:Top-Composer-Loader: filename conf/isolate_components2.js (includer._init_filename() @341)
2017-10-05 12:12:24,885:INFO    :cohorte.config.includer:Top-Composer-Loader: read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/isolate_components2.js (includer._read_contents_file() @263)
2017-10-05 12:12:24,890:INFO    :cohorte.config.includer:Top-Composer-Loader: read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/isolate_components2.js (includer._read_contents_file() @263)
2017-10-05 12:12:24,894:INFO    :cohorte.config.includer:Top-Composer-Loader: _getContent conf/components3.js (includer._get_content() @375)
2017-10-05 12:12:24,895:INFO    :cohorte.config.includer:Top-Composer-Loader: filename conf/components3.js (includer._init_filename() @341)
2017-10-05 12:12:24,907:INFO    :cohorte.config.includer:Top-Composer-Loader: _getContent conf/isolate_components3.js (includer._get_content() @375)
2017-10-05 12:12:24,907:INFO    :cohorte.config.includer:Top-Composer-Loader: filename conf/isolate_components3.js (includer._init_filename() @341)
2017-10-05 12:12:24,911:INFO    :cohorte.config.includer:Top-Composer-Loader: read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/isolate_components3.js (includer._read_contents_file() @263)
2017-10-05 12:12:24,915:INFO    :cohorte.config.includer:Top-Composer-Loader: read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/isolate_components3.js (includer._read_contents_file() @263)
2017-10-05 12:12:24,917:INFO    :cohorte.composer.top.composer:Top-Composer-Loader: Started composition: hello-app -> d510f298-d978-47d0-b207-e072de3ee55c (composer.validate() @136)
2017-10-05 12:12:24,918:INFO    :cohorte.composer.node.top_store:Top-Composer-Loader: Storing top composition d510f298-d978-47d0-b207-e072de3ee55c (top_store.store() @111)
2017-10-05 12:12:24,920:INFO    :debug.debug         :Top-Composer-Loader: Debug REST API validated (api.validate() @770)
2017-10-05 12:12:24,923:INFO    :debug.debug         :Top-Composer-Loader: Bound to /debug (api.bound_to() @783)
2017-10-05 12:12:24,926:INFO    :webadmin.webadmin   :Top-Composer-Loader: Webadmin validated (webadmin.validate() @1127)
2017-10-05 12:12:24,926:INFO    :webadmin.webadmin   :Top-Composer-Loader: Bound to /webadmin (webadmin.bound_to() @1143)
2017-10-05 12:12:38,758:INFO    :cohorte.composer.node.criteria.distance.history:iPOPO-repository-loader: Using neutral candidate for AR (history.vote() @212)
2017-10-05 12:12:38,759:INFO    :cohorte.composer.node.criteria.distance.history:iPOPO-repository-loader: Using neutral candidate for EN (history.vote() @212)
2017-10-05 12:12:38,759:INFO    :cohorte.composer.node.criteria.distance.history:iPOPO-repository-loader: Vote preference for ES: components1 (history.vote() @204)
2017-10-05 12:12:38,760:INFO    :cohorte.composer.node.criteria.distance.history:iPOPO-repository-loader: Vote preference for FR: components1, components2 (history.vote() @204)
2017-10-05 12:12:38,761:INFO    :cohorte.composer.node.criteria.distance.history:iPOPO-repository-loader: Vote preference for HC: components1, components2 (history.vote() @204)
2017-10-05 12:12:38,762:INFO    :cohorte.composer.node.history:iPOPO-repository-loader: Node composer stored in history:
{'components1': ('EN', 'FR'),
 'components2': ('ES',),
 'components3': ('AR',),
 'web': ('HC',)} (history.store() @117)
2017-10-05 12:12:38,765:INFO    :herald.core         :iPOPO-repository-loader: No peer in group all (core.fire_group() @711)
2017-10-05 12:12:38,766:INFO    :cohorte.config.includer:NodeComposer-Starter-0: _getContent conf/python-isolate.js (includer._get_content() @375)
2017-10-05 12:12:38,767:INFO    :cohorte.config.includer:NodeComposer-Starter-0: filename conf/python-isolate.js (includer._init_filename() @341)
2017-10-05 12:12:38,768:INFO    :cohorte.config.includer:NodeComposer-Starter-0: read file /Users/apisu/Documents/tests/cohorte/cohorte-test/conf/python-isolate.js (includer._read_contents_file() @263)
2017-10-05 12:12:38,770:INFO    :cohorte.config.includer:NodeComposer-Starter-0: _getContent conf/python-common-http.js (includer._get_content() @375)
2017-10-05 12:12:38,770:INFO    :cohorte.config.includer:NodeComposer-Starter-0: filename conf/python-common-http.js (includer._init_filename() @341)
2017-10-05 12:12:38,771:INFO    :cohorte.config.includer:NodeComposer-Starter-0: read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/python-common-http.js (includer._read_contents_file() @263)
2017-10-05 12:12:38,772:INFO    :cohorte.config.includer:NodeComposer-Starter-0: read file /Users/apisu/Documents/tests/cohorte/cohorte-test/conf/python-common-http.js (includer._read_contents_file() @263)
2017-10-05 12:12:38,773:INFO    :cohorte.config.includer:NodeComposer-Starter-0: read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/python-common-http.js (includer._read_contents_file() @263)
2017-10-05 12:12:38,775:INFO    :cohorte.config.includer:NodeComposer-Starter-0: _getContent conf/composer/python-isolate.js (includer._get_content() @375)
2017-10-05 12:12:38,775:INFO    :cohorte.config.includer:NodeComposer-Starter-0: filename conf/composer/python-isolate.js (includer._init_filename() @341)
2017-10-05 12:12:38,777:INFO    :cohorte.config.includer:NodeComposer-Starter-0: read file /Users/apisu/Documents/tests/cohorte/cohorte-test/conf/composer/python-isolate.js (includer._read_contents_file() @263)
2017-10-05 12:12:38,794:WARNING :cohorte.monitor.basic:NodeComposer-Starter-0: Missing artifacts: {'os.path'} (basic.start_isolate() @293)
2017-10-05 12:12:44,975:INFO    :cohorte.config.includer:NodeComposer-Starter-0: _getContent conf/python-isolate.js (includer._get_content() @375)
2017-10-05 12:12:44,976:INFO    :cohorte.config.includer:NodeComposer-Starter-0: filename conf/python-isolate.js (includer._init_filename() @341)
2017-10-05 12:12:44,979:INFO    :cohorte.config.includer:NodeComposer-Starter-0: read file /Users/apisu/Documents/tests/cohorte/cohorte-test/conf/python-isolate.js (includer._read_contents_file() @263)
2017-10-05 12:12:44,981:INFO    :cohorte.config.includer:NodeComposer-Starter-0: _getContent conf/python-common-http.js (includer._get_content() @375)
2017-10-05 12:12:44,982:INFO    :cohorte.config.includer:NodeComposer-Starter-0: filename conf/python-common-http.js (includer._init_filename() @341)
2017-10-05 12:12:44,983:INFO    :cohorte.config.includer:NodeComposer-Starter-0: read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/python-common-http.js (includer._read_contents_file() @263)
2017-10-05 12:12:44,985:INFO    :cohorte.config.includer:NodeComposer-Starter-0: read file /Users/apisu/Documents/tests/cohorte/cohorte-test/conf/python-common-http.js (includer._read_contents_file() @263)
2017-10-05 12:12:44,985:INFO    :cohorte.config.includer:NodeComposer-Starter-0: read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/python-common-http.js (includer._read_contents_file() @263)
2017-10-05 12:12:44,990:INFO    :cohorte.config.includer:NodeComposer-Starter-0: _getContent conf/composer/python-isolate.js (includer._get_content() @375)
2017-10-05 12:12:44,990:INFO    :cohorte.config.includer:NodeComposer-Starter-0: filename conf/composer/python-isolate.js (includer._init_filename() @341)
2017-10-05 12:12:44,992:INFO    :cohorte.config.includer:NodeComposer-Starter-0: read file /Users/apisu/Documents/tests/cohorte/cohorte-test/conf/composer/python-isolate.js (includer._read_contents_file() @263)
2017-10-05 12:12:45,494:INFO    :cohorte.config.parser:NodeComposer-Starter-0: load isolate conf file web.js (parser.prepare_isolate() @366)
2017-10-05 12:12:46,735:INFO    :cohorte.config.includer:NodeComposer-Starter-0: _getContent conf/web.js (includer._get_content() @375)
2017-10-05 12:12:46,735:INFO    :cohorte.config.includer:NodeComposer-Starter-0: filename conf/web.js (includer._init_filename() @341)
2017-10-05 12:12:48,702:INFO    :cohorte.config.includer:NodeComposer-Starter-0: _getContent conf/isolate_web.js (includer._get_content() @375)
2017-10-05 12:12:48,703:INFO    :cohorte.config.includer:NodeComposer-Starter-0: filename conf/isolate_web.js (includer._init_filename() @341)
2017-10-05 12:12:48,706:INFO    :cohorte.config.includer:NodeComposer-Starter-0: read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/isolate_web.js (includer._read_contents_file() @263)
2017-10-05 12:12:48,709:INFO    :cohorte.config.includer:NodeComposer-Starter-0: read file /Users/apisu/Documents/repositories/cohorte/cohorte-platforms/cohorte-home/docker/demo/install/node/conf/isolate_web.js (includer._read_contents_file() @263)
2017-10-05 12:13:22,448:INFO    :cohorte.forker.basic:Pelix-Shell-TextConsole: Sending 'forker stopping' signal. (basic._send_stopping() @354)
2017-10-05 12:13:22,449:INFO    :herald.core         :Pelix-Shell-TextConsole: No peer in group all (core.fire_group() @711)
2017-10-05 12:13:22,451:INFO    :webadmin.webadmin   :Pelix-Shell-TextConsole: Unbound from /webadmin (webadmin.unbound_from() @1150)
